src/core/embedders/StandardFontEmbedder.ts

```diff
@@ -21,21 +21,22 @@ export interface Glyph {
  *   https://github.com/foliojs/pdfkit/blob/f91bdd61c164a72ea06be1a43dc0a412afc3925f/lib/font/afm.coffee
  */
 class StandardFontEmbedder {
-  static for = (fontName: FontNames) => new StandardFontEmbedder(fontName);
+  static for = (fontName: FontNames, customName?: string) =>
+    new StandardFontEmbedder(fontName, customName);
 
   readonly font: Font;
   readonly encoding: Encoding;
   readonly fontName: string;
 
-  private constructor(fontName: FontNames) {
+  private constructor(fontName: FontNames, customName?: string) {
     // prettier-ignore
     this.encoding = (
         fontName === FontNames.ZapfDingbats ? Encodings.ZapfDingbats
       : fontName === FontNames.Symbol       ? Encodings.Symbol
       : Encodings.WinAnsi
     );
     this.font = Font.load(fontName);
-    this.fontName = this.font.FontName;
+    this.fontName = customName || this.font.FontName;
```

Comment:
In the `CustomFontEmbedder` I noticed we left the `this.fontName = ...` assignment alone and added a new `customName` class member. And then we determine which one to use when we're computing the value for `BaseFont` with a simple `||`. Can we follow the same approach here for the sake of consistency?

Reply:
@Hopding that seems reasonable. I did notice that in the standard font embedder that `embedIntoContext` references the font directly as `this.font.fontName` rather than the `fontName` class member we set above so I'm not sure it was working correctly anyway.

Reply:
@Hopding that seems reasonable. As it turns out the `embedIntoContext` function in the standard font embedder was actually just referencing the font name directly as `this.font.fontName`, so my change would not have worked as it was anyway.